### ISSUE TITLE
Where are my tests?

### ISSUE THREAD

**Issue by sattvik** [clojure-android/neko#42]
I wrote tests years ago, but now they are all disabled.  We need to develop a strategy to get them working again.

**Comment by alexander-yakushev** [comment]
I left the test/ folder from the original Neko untouched. I just haven't used those tests, so I never checked if they still work.

I agree that we need testing, but it should be device-independent like RoboElectric. Something we can put onto Travis or other CI.

**Comment by sattvik** [comment]
I agree.  I've started thinking about it, but it will take some time before I have a workable solution.

**Comment by alexander-yakushev** [comment]
Renamed for clarity.
